doc/extensibility.md

```diff
@@ -104,7 +114,18 @@ significant wins in terms of performance and payload size.
 Utilizing `MessagePack` for exchanging JSON-RPC messages is incredibly easy.
 Check out the `BasicJsonRpc` method in our [MessagePackFormatterTests][MessagePackUsage] class.
 
+#### When to use `SystemTextJsonFormatter`
+
+When the remote party does not support MessagePack but does support UTF-8 encoded JSON,
+`SystemTextJsonFormatter` offers the most performant choice available.
+
+#### When to use `JsonMessageFormatter`
+
+This formatter is the default for legacy reasons, and offers compatibility with data types that can only be serialized with Newtonsoft.Json.
+It produces JSON text and allows configuring the text encoding, with UTF-8 being the default.
```

Comment:
It looks like if you are not using Newtonsoft attributes and Newtonsoft converters then STJ vs Newtonsoft should be compatible. It may be worth repeating compatibility with DataContract attributes on when to use  SystemTextJsonFormatter section again in case someone directly jumps through this section.

Reply:
Done.